ci/pipeline

```diff
@@ -143,8 +143,46 @@ def run(): (String, Option[awsClient.Artifact]) = {
 @main
 def jenkins(): Unit = {
   if(utils.isPullRequest) {
-    asPullRequest { run() }
+    asPullRequest {
+      val version = build()
+      // Uploads
+      val artifact = uploadTarballToS3(version, "builds")
+      (version, artifact)
+    }
   } else {
-    run()
+    val version = build()
+    uploadTarballToS3(version, "builds")
   }
 }
+
+
+/**
+ * Executes the Community Release which includes:
+ *
+ *  1. tarball with version details 
+ *  Unlike the marathon build, the version details are currently expected to be
```

Comment:
Please avoid the extra spaces after a punctuation. This makes the text a little less legible.